src/app/connect/create/create.ts

```diff
@@ -143,4 +145,15 @@ export class CreateProfileComponent {
       this.error = err.message;
     }
   }
+
+  handleFileInput(event: any) {
+    const file = event.target.files[0];
+    if (file) {
+      const reader = new FileReader();
+      reader.onload = () => {
+        this.profile.picture = reader.result as string;
```

Comment:
This will put the picture on the profile and save as base64, right? I understand next steps will be to upload to IPFS or similar service. We don't want to put base64 encoded images to the profile saved to Nostr, as it will make the profile JSON too large.

Reply:
To optimize profile JSON size, I would store images separately and reference them with URLs or identifiers instead of saving them as base64 strings in the profile.